src/com/facebook/buck/ide/intellij/IjProjectTemplateDataPreparer.java

```diff
@@ -478,6 +479,30 @@ private void addAndroidResourcePaths(
     }
   }
 
+  private void addAndroidCompilerOutputPath(
+      Map<String, Object> androidProperties,
+      Path moduleBasePath,
+      IjModuleAndroidFacet androidFacet) {
+    String variantName = "debug";
```

Comment:
it was a default fallback incase the manifest wasn't present (since it is optional - but it shouldn't happen)